kubernetes/config/samples/imagenet.yaml

```diff
@@ -20,11 +20,8 @@ spec:
             - name: elasticjob-worker
               image: <your_dockerhub_username>/examples:imagenet
               imagePullPolicy: Always
-              ports:
-                - containerPort: 10291
-                  name: elasticjob-port
               args:
-                - "s3://<your_script_path>/petctl/<username>/imagenet_job/main.py"
+                - "/opt/main.py"
```

Comment:
this can be replaced with `/workspace/examples/imagenet/main.py` since you are using `torchelastic/examples` base image